src/item.cpp

```diff
@@ -3661,6 +3661,10 @@ bool item::can_contain( const itype &tp ) const
 
 bool item::spill_contents( Character &c )
 {
+    if( is_container_empty() || !is_container() ) {
```

Comment:
Fair enough, it just reads nicer that way 'if its not a container of it's empty don't do anything'